Board/src/main/java/com/example/board/member/MemberJPARepository.java

```diff
@@ -0,0 +1,16 @@
+package com.example.board.member;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Optional;
+
+public interface MemberJPARepository extends JpaRepository<Member, Integer> {
+
+    @Query("select m from Member as m where m.name = :name")
+    Optional<Member> findByName(String name);
```

Comment:
```suggestion
    @Query("select m from Member as m where m.name = :name LIMIT 1")
    Optional<Member> findByName(String name);
```
아마 JPA가 해결해줄 수 있지만 단건검색은 습관적으로 LIMIT 1을 넣어주시는게 좋습니다.